app/views/admin/supplier_users/new.html.erb

```diff
@@ -0,0 +1,22 @@
+<% content_for :javascript_pack_tags, javascript_pack_tag("admin/supplier-users", defer: true) %>
+
+<% content_for :before_content, govuk_back_link(
+  text: "Back",
+  href: admin_supplier_users_path)
+%>
+
+<h1 class="govuk-heading-l">What supplier does this user belong to?</h1>
+
+<%= form_with model: @supplier_user_form, url: new_admin_supplier_user_path, method: :post do |f| %>
+  <%= f.govuk_collection_select(
```

Comment:
So is this enhanced by autocomplete? Or is it some other field?

Does it work for collection select? :O

Reply:
Yes. I think to most of those questions...

Reply:
Try it out 🙂 https://ecf-review-pr-74.london.cloudapps.digital/admin/suppliers/users/new

Reply:
![image](https://user-images.githubusercontent.com/40488007/106581819-31540d80-653b-11eb-9fb9-076cbecc43b0.png)